packages/icons/plugins/RedPacket.tsx

```diff
@@ -0,0 +1,45 @@
+import { createIcon } from '../utils'
```

Comment:
It looks like it conflicts with [this](https://github.com/DimensionDev/Maskbook/pull/3443/files#diff-f800213e606c38fc4120426314959b69e9a57231aa80d166a9ee6771f80c6bad)